api/src/main/resources/workbench-api.yaml

```diff
@@ -2405,6 +2405,27 @@ paths:
           description: success
           content: {}
       x-codegen-request-body-name: request
+  /v1/admin/users/{userId}/userDisabledEvents:
+    get:
+      tags:
+        - userAdmin
+      description: |
+        List all of the user's disabled/enabled events
```

Comment:
please add the "requires X authority" comment used in other admin endpoints

Reply:
Done